test/CreateTest.php

```diff
@@ -140,6 +146,11 @@ public function test_08_good_credentials_construct($username, $password = null)
 			$this->assertTrue($c->ping());
 		}
 
+		public function test_09_inheritance() {
+			$c = new TarantoolExt('localhost', self::$port);
```

Comment:
```suggestion
			$c = new class ('localhost', self::$port) extends Tarantool {
			    public $property = 42;
			};
```

Reply:
Verified that it catches the error and works on 7.0 and 7.4. Okay. Will apply in the next force-push.